Library/Martial Arts/Martial Arts Skills.skl

```diff
@@ -2280,104 +2313,264 @@
 			}
 		},
 		{
-			"id": "17ece65b-9c42-45c0-9bec-474ccb14b332",
+			"id": "17ece65b-9c42-45c0-9bec-474ccb14b332",
+			"type": "technique",
+			"name": "Low Fighting",
+			"reference": "MA77",
+			"tags": [
+				"Combat",
+				"Melee Combat",
+				"Technique",
+				"Weapon"
+			],
+			"difficulty": "h",
+			"points": 2,
+			"default": {
+				"type": "skill",
+				"name": "@Combat Skill@",
+				"modifier": -2
+			},
+			"limit": 0
+		},
+		{
+			"id": "8b3e81d5-3725-4128-aaf9-60d4e1726f26",
+			"type": "technique",
+			"name": "Low-Line Defense",
+			"reference": "MA77",
+			"tags": [
+				"Combat",
+				"Melee Combat",
+				"Technique",
+				"Weapon"
+			],
+			"difficulty": "h",
+			"points": 1,
+			"default": {
+				"type": "parry",
+				"name": "Boxing",
+				"modifier": -2
+			},
+			"limit": 0
+		},
+		{
+			"id": "bad18fdd-51e5-4a37-b98c-72df44fd4ebd",
+			"type": "technique",
+			"name": "Low-Line Defense",
+			"reference": "MA77",
+			"tags": [
+				"Combat",
+				"Melee Combat",
+				"Technique",
+				"Weapon"
+			],
+			"difficulty": "h",
+			"points": 1,
+			"default": {
+				"type": "parry",
+				"name": "Sumo Wrestling",
+				"modifier": -2
+			},
+			"limit": 0
+		},
+		{
+			"id": "bc7b80e3-ebcd-4dd6-a1ec-ceb9317561eb",
+			"type": "technique",
+			"name": "Lower-Body Arm Lock",
+			"reference": "MA79",
+			"tags": [
+				"Combat",
+				"Melee Combat",
+				"Technique",
+				"Weapon"
+			],
+			"difficulty": "a",
+			"points": 1,
+			"default": {
+				"type": "skill",
+				"name": "Judo",
+				"modifier": -2
+			},
+			"limit": 4
+		},
+		{
+			"id": "edd2e851-5b54-42aa-b879-c74e550bd564",
+			"type": "technique",
+			"name": "Lower-Body Arm Lock",
+			"reference": "MA78",
+			"tags": [
+				"Combat",
+				"Melee Combat",
+				"Technique",
+				"Weapon"
+			],
+			"difficulty": "a",
+			"points": 1,
+			"default": {
+				"type": "skill",
+				"name": "Wrestling",
+				"modifier": -2
+			},
+			"limit": 4
+		},
+		{
+			"id": "fd1658c3-5807-4208-864f-3ac894c2f30a",
+			"type": "technique",
+			"name": "Lower-Body Leg Lock",
+			"reference": "MA79",
+			"tags": [
+				"Combat",
+				"Melee Combat",
+				"Technique",
+				"Weapon"
+			],
+			"difficulty": "a",
+			"points": 1,
+			"default": {
+				"type": "skill",
+				"name": "Judo",
+				"modifier": -2
+			},
+			"limit": 4
+		},
+		{
+			"id": "b6cbca12-ab84-4830-a962-8b34455ae141",
+			"type": "technique",
+			"name": "Lower-Body Leg Lock",
+			"reference": "MA78",
+			"tags": [
+				"Combat",
+				"Melee Combat",
+				"Technique",
+				"Weapon"
+			],
+			"difficulty": "a",
+			"points": 1,
+			"default": {
+				"type": "skill",
+				"name": "Wrestling",
+				"modifier": -2
+			},
+			"limit": 4
+		},
+		{
+			"id": "12ba82ac-9567-49b4-9358-ad78fd632b02",
 			"type": "technique",
-			"name": "Low Fighting",
+			"name": "Mounted Shooting",
```

Comment:
Mounted Shooting does have Riding or a vehicle-operation skill as a prerequisite.

Reply:
I wasn't quite sure what to do in this case.
I assumed there to be way too many Vehicle skills to be practical to list them all in the prerequisites. For example, I assumed that each specialization of Driving would be counted as a separate skill in GCS (fortunately not the case).
I just checked out GURPS Skill Categories and there's only 14 actual Vehicle skills. A lot, but definitely doable. So I'll just get on that real quick.

Reply:
So it turns out even less of those were actually involved with operating a vehicle, so that's fortunate.